projects/storyteller/webapp/src/app/components/feed/feed.component.html

```diff
@@ -1,4 +1,4 @@
-<div *ngIf="!storylist">
+<div *ngIf="!storylist && !screenshots">
```

Comment:
You could create a boolean variable `isLoading`, which is `true` by default.
And make it `false`, when your page is loaded.